boxoffice/assets/js/models/util.js

```diff
@@ -28,6 +28,14 @@ export const Util = {
     else {
       return moment(dateTimeString).toString();
     }
+  },
+  getElementId: function(htmlString) {
+    return htmlString.match(/id="(.*?)"/)[1];
+  },
+  getFormTemplate: function(htmlString) {
+    // Add on click event handler for Ractive to submit the form
+    return `${htmlString.slice(0, htmlString.search(/type="submit"/))} on-click="onFormSubmit(event)" 
+      {{#formOnSubmit}}disabled{{/}} ${htmlString.slice(htmlString.search(/type="submit"/))}`;
```

Comment:
Where is `formOnSubmit` coming from?

Reply:
Do you mean it is better to pass the submit event handler as an argument to `getFormTemplate`?

Reply:
Yes. A new developer shouldn't have to understand the internals of this function to use it. The inputs, outputs and side effects (if any) should be obvious.

Reply:
Fixed